test/FixCode/fixits-apply-all.swift

```diff
@@ -2,7 +2,9 @@
 // The particular diagnostics used are not important.
 
 // RUN: %swift -parse -target %target-triple %s -fixit-all -emit-fixits-path %t.remap
-// RUN: c-arcmt-test %t.remap | arcmt-test -verify-transformed-files %s.result
+// RUN: c-arcmt-test %t.remap > %t.c-arcmt-test.out
+// RUN: arcmt-test -verify-transformed-files %s.result < %t.c-arcmt-test.out
+// RUN: sed -n 0~2p %t.c-arcmt-test.out | xargs rm -rf
```

Comment:
I'm concerned about the `sed ... | xargs rm -rf` pipeline.  Even though it is correct now, the chances are non-zero that something might go wrong in the sed part and some important data will be deleted (e.g., if '/' appears in its output).